### ISSUE TITLE
Multi-Monitor curruption

### ISSUE THREAD

**Issue by KPreston2900050** [clsid2/mpc-hc#2010]
This is the only video player I have used for as long as I can remember but as soon as I went to Windows 11 and a Multi-monitor environment last year this is the only thing that has given me issues.

When dragging the running video from my Primary to the extended monitor the video crashes.
It does not happen with audio and I suspected that it has to do with changing monitor resolutions so I tried it with two of the same monitors and it still crashed.

After I move it and crashed all I have to do is press stop then restart. It then plays fine.
I tried other video players and the crashing between two monitors does not happen.

Windows 11 22H2
MPC-HC 64 v2.0.05
Build date Jan 22, 2023
LAV Filters 0.77.1.2

**Comment by clsid2** [comment]
Which video renderer?

**Comment by KPreston2900050** [comment]
I only use MAD DOG

     On Sunday, February 5, 2023, 07:07:07 AM EST, clsid2 ***@***.***> wrote:  
 
 


Which video renderer?

**Comment by clsid2** [comment]
MadVR is currently not working very well on Windows 11. Use MPC Video Renderer instead.

**Comment by KPreston2900050** [comment]
OK, so I changed to MPC but now when I play a video it resets my screen mode to HDR, flashes everything and ruins my Icons positions.

I forgot how I disabled HDR in MadVR but it would be great if there was a feature to turn that crap off in the menu.

I even get annoyed when it pops up with screen some overlay telling me what screen resolution it is.
I know what resolution because I put it there.
Just play the movie, stop with the feedback and changing settings. Just Playyyyyyyyyyyyyy.

My computer has to do crap for me, my car has to tell me to drive safe and put on a seat belt, my microwave over has to beep 50 times....YOUR FOODS DONE....YOUR FOODS DONE....My &&&&ing Dryer has to whistle out a tune just like my damn Dish Washer.

Dude....I hate the 21st century and working on the bridge of the USS Enterprise would make you kill yourself.


Just play the movie, who asked for anything else.

     On Sunday, February 5, 2023, 05:20:39 PM EST, clsid2 ***@***.***> wrote:  
 
 


MadVR is currently not working very well on Windows 11. Use MPC Video Renderer instead.
